src/main.test.js

```diff
@@ -54,35 +54,35 @@ test("Valid input with comment,middle of line", () => {
 test("Valid input with comment, start of line", () => {
   const testEnv =
     "#DATABASE_URL: email = test@mail.com\nDATABASE_URL: email = test@mail.com";
-  expect(envSpecToHTML(testEnv)).toEqual(
+  expect(envSpec.parse(testEnv)).resolves.toEqual(
```

Comment:
You need to `return` this, in order to mark the test successfully as passed/failing ([see example](https://jestjs.io/docs/en/expect.html#resolves)).